dev/Dockerfile.test.dockerignore

```
@@ -0,0 +1,24 @@
+.git
+mlruns
+docs
+apidocs
+mlflow.Rcheck
+outputs
+
+dev
+# required for Docker build
+!requirements/test-requirements.txt
+!requirements/lint-requirements.txt
+!requirements/extra-ml-requirements.txt
+
+node_modules
```

```suggestion
*/node_modules
```

`node_modules` doesn't seem to exclude `node_modules` in `mlflow/server.js`.

---

Or we can ignore everything except files that are required while building the image.

```
**

!requirements
!setup.py
!mlflow/version.py
!README.rst
```